RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 
 21.0
 -----
+* [***] [internal] Updates the target sdk to 31 - Android 12 [https://github.com/wordpress-mobile/WordPress-Android/pull/17153]
```

Comment:
Why is it internal?

Reply:
@ravishanker Since this PR only changes the `targetSdkVersion` to 31 and doesn't contain any user-facing changes. I thought there was no need for any release notes. Only to get extra visibility for beta testing, as confirmed by @thehenrybyrd I added this note.